test/openAPI/features/wallet_generate_link_code.feature

```diff
@@ -0,0 +1,24 @@
+@method=POST @endpoint=/linked-authorization/link-code
+Feature: The endpoint to generate link code.
+
+  @smoke @unit @positive
+  Scenario: Successfully generate link code smoke type test
+    Given Wants to generate link code
+    When Send POST /linked-authorization/link-code request with given X-XSRF-TOKEN header, transactionId and requestTime
```

Comment:
Hope we are also sending the cookie with same value as XSRF-TOKEN

Reply:
this is just a state-less cookie, we don't need any sessions here. Just a new cookie with key as 'XSRF-TOKEN' and value as same given in the X-XSRF-TOKEN request header is enough

Reply:
Changes are available in https://github.com/GovStackWorkingGroup/bb-identity/pull/42

Reply:
cookies key should be 'XSRF-TOKEN' and request header should have 'X-XSRF-TOKEN', there is a small difference in the key

Reply:
moving comment to new PR and marking this comment as resolved